moveit_core/planning_interface/include/moveit/planning_interface/planning_response.h

```diff
@@ -38,9 +38,9 @@
 #define MOVEIT_PLANNING_INTERFACE_PLANNING_RESPONSE_
 
 #include <moveit/robot_trajectory/robot_trajectory.h>
-#include <moveit_msgs/MoveItErrorCodes.h>
-#include <moveit_msgs/MotionPlanResponse.h>
-#include <moveit_msgs/MotionPlanDetailedResponse.h>
+#include <moveit_msgs/msg/moveIt_error_codes.hpp>
```

Comment:
I believe that this is wrong. With the camelcase translation it should be:
```suggestion
#include <moveit_msgs/msg/move_it_error_codes.hpp>
```

Reply:
@mlautman  I was puzzled by this too, so moveIt is not considered a single word?

Reply:
It isn't. The auto message generation assumes that the msg names use camel case and then converts it to underscores to create the message header files. This results in a translation from `MoveItSomeMsg.msg` -> `move_it_some_msg.hpp`

Reply:
ah okay :+1: 
I've fixed those and pushed a new commit.

Reply:
@mlautman